lib/Platform/Intl/java/com/facebook/hermes/intl/DateTimeFormat.java

```diff
@@ -124,6 +127,17 @@ private Object ToDateTimeOptions(Object options, String required, String default
       }
     }
 
+    if (!JSObjects.isUndefined(JSObjects.Get(options, "dateStyle")) || !JSObjects.isUndefined(JSObjects.Get(options, "timeStyle")) )
+      needDefaults = false;
+
+    if (required.equals("date") && !JSObjects.isUndefined(JSObjects.Get(options, "timeStyle"))) {
+      // throw // TODO throw TypeError
```

Comment:
can you resolve this todo?

Reply:
This check is added in the native code .. I initially added this TODO in the java code to not forget adding checks to native code ..
Currently we don't have the mechanism to throw the validation errors from Java code .. all such validations must be done in C++ code today..